src/main/resources/static/templates/CSRFVulnerability/LEVEL_2/CSRFVulnerability.js

```
@@ -0,0 +1,29 @@
+function addingEventListenerToSendButton() {
+  document.getElementById("sendBtn").addEventListener("click", function () {
+    let url = getUrlForVulnerabilityLevel();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'url' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)